### DISCUSSION TITLE
LTSP server with  pFsense vlans

### DISCUSSION THREAD

**Discussion by doguibnu** [ltsp/ltsp#708] [General]
Hello @alkisg 
How are you?

My scenario (to remember the chat conversation)
pFsense with 107 vlans (10.10.10.x, 10.10.20.x, 10.10.30.x and more)
At my first lan (10.1.1.x) after test, the client machine UP nice!
pFsense with DHCP enable mapping mac address, right

As you tell me at the last time on chat room that LTSP server side I need add vlans IPs that I want. For example: In Network Connection add vlans IPs following:
10.10.20.x
10.10.30.x
10.10.40.x
save 

and run the command:
# ltsp dnsmasq

Right but, there is something not well here, because after testing in out place (that use 10.10.20.x vlan IP) the client machine no boot and UP. The client side stay to find connection. But, we can see (appear) the LTSP server send or work for the client side, because we can see at pfsense list. 

Please, Can you help me?
Thank you

Douglas

**Comment by alkisg** [comment]
Hi Douglas, this issue isn't related to the LTSP code base, it's essentially a sysadmin/pfsense/networking task.
I'm sorry but currently my paid works do not leave me enough time to help you with this.
It's best to direct your question at @everyone instead of just @alkisg, and see if some other member of the LTSP or pfsense or networking communities is able to help you.
